config/config.go

```diff
@@ -263,9 +263,10 @@ type Config struct {
 	// to support the upgrade process. They can be removed in future.
 
 	// EnableBatchDML, MemQuotaQuery, OOMAction unused since bootstrap v90
-	EnableBatchDML bool   `toml:"enable-batch-dml" json:"enable-batch-dml"`
-	MemQuotaQuery  int64  `toml:"mem-quota-query" json:"mem-quota-query"`
-	OOMAction      string `toml:"oom-action" json:"oom-action"`
+	EnableBatchDML bool       `toml:"enable-batch-dml" json:"enable-batch-dml"`
+	MemQuotaQuery  int64      `toml:"mem-quota-query" json:"mem-quota-query"`
+	OOMAction      string     `toml:"oom-action" json:"oom-action"`
+	TrxSummary     TrxSummary `toml:"transaction-summary" json:"transaction-summary"`
```

Comment:
We should move it. There is a line above it saying "The following items are deprecated.".